academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+* **IBC/APP:** a standard defining an inter-blockchain communication system application-layer protocol.
+
+Assume we have two chains: **chain *A* and chain *B***. Both will implement the ICS/TAO for IBC.
+
+![IBC overview - two connected chains](images/ibcoverview.png)
+
+Notice three crucial elements in the diagram:
+
+* The chains depend on relayers to communicate.
+* Many relayers can serve one or more channels to send messages between the chains.
+* Each side of the relay uses the light client of the other chain to quickly verify incoming messages.
+
+### IBC/TAO - transport layer
+
+The transport layer (TAO) provides the necessary infrastructure to establish secure connections and authenticate data packets between chains. The application layer defines exactly how data packets should be packaged and interpreted by the sending and receiving chains.
+
+The great promise of IBC is the possibility of optimizing not only on the transport layer but also on the layers above it: a powerful and secure but ultimately generic and permissionless transport layer, which supports any number of diverse and innovative applications from asset transfers and oracle data provision to decentralized identities and cross-chain, synchronous transaction validation built on top of it.
+
+![The transport layer](images/tao.png)
+
+In the diagram, the relationship between the ICS definitions in the category TAO are illustrated - the arrows illustrating the requirements.
+
+Simply put, the transport layer includes:
+
+* Clients - ICS [2](https://github.com/cosmos/ibc/tree/master/spec/core/ics-002-client-semantics), [6](https://github.com/cosmos/ibc/tree/master/spec/client/ics-006-solo-machine-client), [7](https://github.com/cosmos/ibc/tree/master/spec/client/ics-007-tendermint-client), [5](https://github.com/cosmos/ibc/tree/master/spec/client/ics-008-wasm-client), [9](https://github.com/cosmos/ibc/tree/master/spec/client/ics-009-loopback-client), [10](https://github.com/cosmos/ibc/tree/master/spec/client/ics-010-grandpa-client)
+  In IBC, the algorithm utilized to verify the consensus transcript and state sub-components of another machine is referred to as a "validity predicate". Pairing the validity predicate with a state the verifier assumes to be correct forms the _light client_ (often shortened to _client_). In the IBC protocol, an actor, which may be an end-user, an off-chain process, or a machine, needs to be able to verify updates to the state of another machine to which the other machine's consensus algorithm has agreed to, and reject any possible updates which the other machine's consensus algorithm has not agreed to. A light client is an algorithm with which a machine can do so. The standard formalizes the light client model and requirements in a way that the IBC protocol can easily integrate with new machines running new consensus algorithms - as long as the associated light client algorithms fulfill the listed requirements.
```

Comment:
I believe this language is from the spec but I felt it's a bit heavy. Maybe we can take the same text that was previously used in the IBC clients section? https://tutorials.cosmos.network/academy/2-main-concepts/ibc.html#clients